formatsvg.py

```diff
@@ -394,22 +373,6 @@ def removePNGs(self):
             except:
                 print("Unable to delete", filePath)
 
-    def manuallyDownloadFlags(self, alpha3CodeFlag1=None, alpha3CodeFlag2=None):
```

Comment:
This method shouldnt be deleted, it can be useful to create custom mashups and not only random ones.

Reply:
Will add it again. What about `getEmoji` (line 61) and `getImage` line(69)?

Reply:
Unused, good catch 👍